go.mod

```diff
@@ -5,11 +5,11 @@ go 1.13
 require (
 	github.com/go-gorp/gorp v2.2.0+incompatible // indirect
 	github.com/golang/mock v1.4.4
-  github.com/hashicorp/go-multierror v1.1.0
+	github.com/hashicorp/go-multierror v1.1.1
 	github.com/magefile/mage v1.11.0
 	github.com/magiconair/properties v1.8.4 // indirect
 	github.com/mattermost/gorp v2.0.1-0.20200527092429-d62b7b9cadfc+incompatible // indirect
-	github.com/mattermost/mattermost-server/v5 v5.3.2-0.20210317142801-5706df9fe0e0
+	github.com/mattermost/mattermost-server/v5 v5.3.2-0.20210413212722-ac76e7b1f4ec
```

Comment:
I can't find this commit in `mattermost-server`'s `master` branch, can we please update it before merging?

Reply:
Yes, I am waiting for https://github.com/mattermost/mattermost-server/pull/17372 and https://github.com/mattermost/enterprise/pull/970 to get merged first.